### ISSUE TITLE
What land-sea mask do we use?

### ISSUE THREAD

**Issue by ccarouge** [ACCESS-Community-Hub/Land-ancillary-creation#7]
CABLE takes the assumption the met. forcing defines the grid and the land-sea mask. This means the simulation runs exactly on the grid of the met. forcing and this forcing needs no interpolation/filling etc. operation.

JULES takes a different approach where the land-sea mask comes from a separate file or the land cover.

Which approach do we want to adopt? It seems avoiding regridding the met forcing before simulation is better but we need to decide.

**Comment by matlipson** [comment]
I agree it will be better to avoid regridding met-forcing and adapting new ancils to match. We can use tools such as [ants.analysis.make_consistent_with_lsm()](https://code.metoffice.gov.uk/doc/ancil/ants/1.0/_modules/ants/analysis.html#make_consistent_with_lsm) to a) remove land data outside met forcing region, b) fill missing land points ([ants.analysis.FillMissingPoints](https://code.metoffice.gov.uk/doc/ancil/ants/1.0/_modules/ants/analysis/_merge.html#FillMissingPoints)) using a "spiral-search" nearest neighbour approach.
